CONTRIBUTING.md

```diff
@@ -67,37 +124,45 @@ keep the code consistent and avoid common pitfalls.
 
 To start working on a patch:
 
- * `git fetch upstream`
- * `git checkout upstream/master -b <name_of_your_branch>`
- * Hack away.
- * Verify changes with [flutter_plugin_tools](https://pub.dartlang.org/packages/flutter_plugin_tools)
-```
-pub global activate flutter_plugin_tools
-pub global run flutter_plugin_tools format --plugins plugin_name
-pub global run flutter_plugin_tools analyze --plugins plugin_name
-pub global run flutter_plugin_tools test --plugins plugin_name
+1. `git fetch upstream`
+2. `git checkout upstream/master -b <name_of_your_branch>`
+3. Hack away!
+
+Once you have made your changes, ensure that it passes the internal analyzer & formatting checks. The following
+commands can be run locally to highlight any issues before committing your code:
+
+```bash
+# Run the analyze check
+melos run analyze
+
+# Format code
+melos run format
 ```
- * `git commit -a -m "<your informative commit message>"`
- * `git push origin <name_of_your_branch>`
+
+Assuming all is successful, commit and push your code:
+
+1. `git commit -a -m "<your informative commit message>"`
+2. `git push origin <name_of_your_branch>`
 
 To send us a pull request:
 
-* `git pull-request` (if you are using [Hub](http://github.com/github/hub/)) or
+- `git pull-request` (if you are using [Hub](http://github.com/github/hub/)) or
   go to `https://github.com/FirebaseExtended/flutterfire` and click the
   "Compare & pull request" button
 
 Please make sure all your checkins have detailed commit messages explaining the patch.
 
-For pull requests that affect only one Flutterfire plugin, use a title that starts
-with the name of the plugin in brackets (e.g. [cloud_firestore]).
+When naming the title of your pull request, please follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0-beta.4/)
+guide. For example, for a fix to the Cloud Firestore plugin:
+
+`fix(cloud_firestore): Fixed a bug!`
```

Comment:
```suggestion
`fix(cloud_firestore): fixed a bug!`
```

Also could we use a different plugin in the example given the potential upcoming rename of cloud_firestore, one less thing to rename later or not forget about?